src/main/java/htsjdk/variant/vcf/VCFFilterHeaderLine.java

```diff
@@ -58,12 +62,21 @@ public VCFFilterHeaderLine(String name) {
      * @param line      the header line
      * @param version   the vcf header version
      */
-    public VCFFilterHeaderLine(String line, VCFHeaderVersion version) {
+    public VCFFilterHeaderLine(final String line, final VCFHeaderVersion version) {
         super(line, version, "FILTER", Arrays.asList("ID", "Description"));
     }
 
     @Override
     public boolean shouldBeAddedToDictionary() {
         return true;
     }
+    
+    /**
+     * get the "Description" field
+     * @return the "Description" or the Filter ID if the description is null or empty 
+     */
+    public String getDescription() {
+        final String desc = getGenericFieldValue("Description");
+        return ( StringUtil.isBlank(desc) ? getID() : desc );
```

Comment:
I think that this logic should not be in htsjdk. getDescription() should return null or blank if there's no decription. the user can choose what to do with that information.